service.sh

```
@@ -2,6 +2,11 @@
 
 # SuperMario Tweaker
 
+# Wait till boot completes 
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)